apps/src/music/views/patternAiPanel.module.scss

```diff
@@ -1,5 +1,7 @@
 @import "color.scss";
 
+$ai-rubric-cyan-hover: #37e1db;
```

Comment:
Should we call this something else music lab/AI-specific since it's declared within the pattern AI css module?

Reply:
Good question.  I think it's worth capturing the relationship to the system color (`$ai-rubric-cyan`).